app/Client/LocalPublisherBannerClassifier.php

```diff
@@ -38,24 +38,25 @@ public function __construct(ClassifierInterface $classifier)
         $this->classifier = $classifier;
     }
 
-    public function fetchBannersClassification(array $bannerUuids): Collection
+    public function fetchBannersClassification(array $bannerIds): Collection
     {
-        $ids = NetworkBanner::findIdsByUuids($bannerUuids);
         $collection = new Collection();
-        foreach ($ids as $bannerId) {
+        $bannersInternalIds = NetworkBanner::findIdsByUuids($bannerIds);
+
+        foreach ($bannersInternalIds as $bannerId) {
             try {
                 $classificationCollection = $this->classifier->fetch($bannerId);
 
                 /** @var Classification $classification */
                 foreach ($classificationCollection as $classification) {
                     $collection->addClassification(
-                        $bannerId,
+                        array_search($bannerId, $bannersInternalIds, true),
```

Comment:
$bannerId is one of $bannersInternalIds, so what is the reason behind this call?

Reply:
`array_search` was removed regarding the fact `publicId` exists as a key :)